civiform/civiform
# Show modal for duplicate submissions

### Description

Show modal for duplicate submissions in North Star.

[View Mocks](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=4067-6334&m=dev)

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes

- [x] Manually tested at 200% size (discovered a general issue https://github.com/civiform/civiform/issues/8710)

### Instructions for manual testing

1. As applicant, submit an application
2. Apply to the same program
3. Go to the review page
4. Click "Submit"
5. Observe the duplicate submission modal

### Issue(s) this completes

Fixes #7995
